test/summarize_tests.jl

```diff
@@ -13,6 +13,7 @@ using Statistics: std
 
     @test 0.48 < parm_df[:a, :mean][1] < 0.52
     @test names(parm_df) == [:parameters, :mean, :std, :naive_se, :mcse, :ess, :rhat]
+    @test parm_df[:a, :] == parm_df[:a]
```

Comment:
Maybe check also explicitly that it does not return other parameters as well? Otherwise it could be that both implementations are incorrect, couldn't it?

Reply:
We check the other implementations down below, like this one, for example:

```julia
@test three_parms_df_2[:, :parameters] == [:a, :b, :g]
```

Reply:
No, nevermind, that indexing is done at chain level. Give me a sec to add that too.